src/templates/base.html

```diff
@@ -143,21 +152,14 @@
         document.addEventListener('DOMContentLoaded', function() {
           const search_bar = document.getElementById('search-bar');
           const query_type = document.getElementById('query-type');
-          const query_input = document.getElementById('query-input');
-
-          //do nothing if search bar is empty
-          search_bar.addEventListener("submit", function() {
-            event.preventDefault();
-            if (query_input.value.trim() !== '') {
-              search_bar.submit();
-            }
-          });
 
           query_type.addEventListener("change", function() {
             if (query_type.value == "games") {
               search_bar.action = "{% url 'game-search-results' %}";
             } else if (query_type.value == "users") {
-              search_bar.action = "{% url 'users:user-search-results' %}";
+              search_bar.action = "{% url 'home' %}"; // placeholder
```

Comment:
Are the placeholders here necessary?

Reply:
This is code from #309. I don't think the placeholder is necessary, so I will remove it